app/templates/instruction-forms/first-degree-assault-form.njk

```diff
@@ -1,2 +1,15 @@
 <input type="hidden" name="first_degree_assult" value="true" />
 <h1>FIRST DEGREE ASSAULT</h1>
+<div class="form-group">
+	<label> Deceased First name: <label>
+	<input type="text" name="dfname">
+</div>
+<div class="form-group">
+	<label> Deceased Last name: <label>
+	<input type="text" name="dlname">
+</div>
+<div class="form-group">
+    <label>Assault executed by?</label><br>
+	<input type="checkbox" name="Firearm" value="drugs"> Use of Firearm<br>
```

Comment:
Even if we leave this as it is we should change the `value` of these to be valid to the option. So in this case the value would be `firearm` instead of `drugs`.

Reply:
this was a silly mistake :D I missed it